spec/facility_spec.rb

```diff
@@ -2,25 +2,275 @@
+      @facility_1.register_vehicle(cruz)
+
+      expect(@facility_1.registered_vehicles).to eq([cruz])
+    end 
+  
+    it 'can give a date a vehicle is registered' do
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+
+            
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(cruz)
+
+      expect(@facility_1.registered_vehicles.first.registration_date).to eq(Date.today)
+    end
+  end 
+  
+  describe '#create_plate' do 
+    it 'can give a plate type :regular' do 
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+
+      @facility_1.create_plate(cruz)
+
+      expect(cruz.plate_type).to eq(:regular)
+    end 
+
+    it 'can give a plate type :ev' do
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+
+      @facility_1.create_plate(bolt) 
+      
+      expect(bolt.plate_type).to eq(:ev)
+    end 
+  end 
+
+  describe '#collected_fees' do 
+    it 'starts with 0 collected fees' do 
+      expect(@facility_1.collected_fees).to eq(0)
+    end 
+
+    it 'collects 100 for regular cars' do 
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      
+            
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(cruz) 
+
+      expect(@facility_1.collected_fees).to eq(100)
+    end 
+
+    it 'collects 200 for ev cars' do 
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      
+            
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(bolt) 
+
+      expect(@facility_1.collected_fees).to eq(200)
+    end 
+  end 
+
+  describe '#register_fee(vehicle)' do 
+    it 'collects 100 when registering car with :regular plates' do 
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      
+      @facility_1.create_plate(cruz)
+      @facility_1.register_fee(cruz)
+
+      expect(@facility_1.collected_fees).to eq(100)
+    end 
+
+    it 'collects 200 when registering car with :ev plates' do
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+
+      @facility_1.create_plate(bolt)
+      @facility_1.register_fee(bolt)
+
+      expect(@facility_1.collected_fees).to eq(200)
+    end 
+  end 
+
+  describe '#administer_written_test(person)' do 
+    it 'does not administer a written if service is not provided' do 
+      expect(@facility_1.services).to eq([])
+
+      registrant_1 = Registrant.new('Bruce', 18, true )
+
+      @facility_1.administer_written_test(registrant_1)
+
+      expect(@facility_1.administer_written_test(registrant_1)).to eq(false)
```

Comment:
good testing that it does NOT administer this test. I'm wondering if this test passed though? It looked like your code didn't actually ever return a `false` value.

Reply:
Yes my test did pass and i was able to return a false value!